content/issues/2023-11-01-cron.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Notification: Significant changes to cron"
+date: "2023-11-01 23:00:00Z"
```

Comment:
Needs date updating to be in the near future of merge time.

Reply:
Changed date to today